templates/BuildMyCharm/config.xml

```diff
@@ -62,13 +63,31 @@ juju add-model test-{{charmname}}-$BUILD_NUMBER
 
 if [ -z "{{referencebundle}}" ]
 then
-    bundletester -e {{controller}}:test-{{charmname}}-$BUILD_NUMBER \
-      -vFt /tmp/$OUTPUT_DIR/{{charmname}}
+    rm -f totest.yaml
+    echo "bundle: /tmp/bundles/{{bundle_fname}}" &gt;&gt; totest.yaml
+    echo "bundle_name: {{charmname}}" &gt;&gt; totest.yaml
+    cwr {{controller}}:test-{{charmname}}-$BUILD_NUMBER totest.yaml --results-dir /srv/artifacts --test-id $BUILD_NUMBER
```

Comment:
This should be gives me : cwr lxd:test-apache-tomee-3 totest.yaml --results-dir /srv/artifacts --test-id 3
usage: cwr [-h] [--result-output RESULT_OUTPUT] [-t TESTDIR] [-b BUNDLE]
           [-d DEPLOYMENT] [--no-destroy] [-l LOG_LEVEL] [-n] [-v] [-F] [-s]
           [-x EXCLUDE] [-y TESTS_YAML] [--test-pattern TEST_PATTERN]
           [--juju-major-version JUJU_MAJOR_VERSION] [--test-id TEST_ID]
           controller [controller ...] test_plan
cwr: error: unrecognized arguments: --results-dir /srv/artifacts

Should it be "--result-output"?